acs-integration-tests/src/test/java/com/ge/predix/integration/test/PolicyEvaluationWithAttributeConnectorIT.java

```diff
@@ -225,30 +213,25 @@ void afterClass() throws IOException {
     }
 
     @Test(dataProvider = "adapterStatusesAndResultingEffects")
-    public void testPolicyEvaluationWithAdapters(final boolean adapterActive, final Effect effect) throws Exception {
-        HttpHeaders httpHeaders = this.getHeadersWithAcsZoneId();
```

Comment:
Since the zone ID doesn't change throughout the test, it's more efficient to instantiate the zone headers once and use them instead of constructing multiple instances by calling `zoneHeader()` multiple times